.github/workflows/govcloud.yml

```diff
@@ -126,3 +126,9 @@ jobs:
           name: accessibility-test-report
           path: /home/runner/work/funding-service-design-frontend/funding-service-design-frontend/axe_reports/*.html
           retention-days: 5
+      - name: ZAP Scan
+        uses: zaproxy/action-full-scan@v0.3.0
+        with:
+          target: ${{ secrets.FORMS_SERVICE_HOST }}
+          allow_issue_writing: False
+          fail_action: true
```

Comment:
ZAP Scan added to run against forms service host to confirm OWASP security passing, with `fail_action: true` flag to ensure process exits on alerts

Reply:
To allow development with XGovFormBuilder, `fail_action: true` has been removed for now and a new bug/issue raised on this repo [Issue #28](https://github.com/communitiesuk/funding-service-design-frontend/issues/28), a bug/ticket to fix this has also been raised in JIRA [#FS-597](https://digital.dclg.gov.uk/jira/browse/FS-597)